js/src/dapps/registry/actions.js

```diff
@@ -1,8 +1,11 @@
 import registryAbi from './abi/registry.json';
 const { api } = window.parity;
-import * as lookup from './Lookup/actions.js';
+import * as accounts from './accounts/actions.js';
+import * as lookup from './lookup/actions.js';
+import * as events from './events/actions.js';
+import * as register from './register/actions.js';
 
-export { lookup };
+export { accounts, lookup, events, register };
 
 export const setContract = (contract) => ({ type: 'set contract', contract });
```

Comment:
Why not follow FSA instead?
https://github.com/acdlite/flux-standard-action

Pros: 
- potential integration with 3rd party libraries
- format predictability (esp. in terms of actions indicating errors/failures)